test/outtest3.F90

```diff
@@ -0,0 +1,166 @@
+! This is a test for NCEPLIBS-bufr.
+!
+! Writes test file 'testfiles/OUT_3' using ISETPRM and IGETPRM, using EXITBUFR with multiple allocations,
+! and using 2-22, 2-36 and 2-37 operators.
+!
+! J. Ator, 2/17/2023
+program outtest3
+  implicit none
+
+  integer*4 isetprm, igetprm
+
+  integer ii, nlv
+
+  real*8 r8vals ( 11, 4 ), r8bitmap ( 26 )
+
+  print *, 'Testing writing OUT_3 using ISETPRM and IGETPRM, using EXITBUFR with multiple allocations,'
+  print *, 'and using 2-22, 2-36 and 2-37 operators.'
+
+#ifdef KIND_8
```

Comment:
Are there any tests where we have to change the settings for KIND_d?

Reply:
No, because both _d and _4 use 4-byte integers.

The only difference in these two builds is that the former uses 8-byte reals instead of 4-byte reals.  But unless we missed something, every real call parameter is already explicitly passed as real*8 anyway in the library specs, so the two builds are now basically equivalent from a library interface perspective.